setup.py

```diff
@@ -47,6 +48,7 @@
 IS_LIN = False
 
 dal_root = os.environ.get("DALROOT")
+n_threads = int(os.environ.get("NTHREADS", os.cpu_count() or 1))
```

Comment:
Do we need this for SHAP?

Reply:
No, it is going into a separate PR https://github.com/intel/scikit-learn-intelex/pull/1553/files